tests/measurement_utility_test.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+# #############################################################################
+# Simple shell script to 'test' that the measurement generated by the
```

Comment:
New test script to validate that the `--policy-key` argument to `embed_policy_key.exe` is working correctly.

During dev, I had made an error in the program, not including the 2nd file's contents. This went undiscovered till something else broke in the simple_app dev steps. Hence, I added this little test-case to protect us from such silly mistakes in the future.